js/ui/grid_core/ui.grid_core.editor_factory.js

```diff
@@ -449,6 +449,7 @@ var EditorFactoryController = modules.ViewController.inherit((function() {
         createEditor: function($container, options) {
             options.cancel = false;
             options.editorElement = $container;
+            options.parentTabIndex = this.option("tabIndex");
```

Comment:
The new parentTabIndex option is not necessary. You can assign tabIndex to options from widget's options if options.tabIndex is not defined,